doc/develop/fixing_data.rst

```diff
@@ -434,6 +454,106 @@ explained in :ref:`extra_facets`, and which content is :download:`available here
 </../esmvalcore/_config/extra_facets/ipslcm-mappings.yml>`. These multi-variable
 files must also undergo some data selection.
 
+.. _how_to_fix_native_datasets:
+
+Add support for new native datasets
+-----------------------------------
+
+This section describes how to add support for additional native datasets.
+You can choose to host this new data source either under a dedicated project or
+under project ``native6``.
+
+To allow ESMValCore to locate the data files, use the following steps:
+
+   - If you want to use the ``native6`` project (recommended for native
```

Comment:
I would say that this is recommended if you can easily move the files to the usual `native6` directory structure, while a custom project is needed if you cannot control the location of the files.